src/main/java/frc/robot/subsystems/drive/SwerveModuleIOTalonFX.java

```diff
@@ -101,7 +101,6 @@ public void config() {
         canCoderConfiguration.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
         turnEncoder.getConfigurator().apply(canCoderConfiguration);
 
-        // TODO: check these gains, or just re-tune
```

Comment:
Not sure why this TODO was removed either. These gains need to be checked and/or retuned.